app/assets/stylesheets/components/_background.scss

```diff
@@ -3,6 +3,11 @@
 .bg-lighter-blue { background-color: $blue-lighter; }
 .bg-lightest-blue { background-color: $blue-lightest; }
 .bg-lightest-red { background-color: $red-lightest; }
+// scss-lint:disable ImportantRule
+.sm-display-inline-block { display: inline-block !important; }
+// scss-lint:enable ImportantRule
+
+.bg-white { background-color: $white !important; }
```

Comment:
Per Code Climate being upset about `!important`, it appears that the gray-ish background is coming from BassCSS, which I expect we'll eventually remove. Since it's the only instance of a readonly field that I can tell, maybe we can override it in `_form.scss` until we remove the BassCSS module?

```scss
.field:read-only:not(select) {
  background-color: $white;
}
```